financial-connections/src/main/java/com/stripe/android/financialconnections/ui/HandleClickableUrl.kt

```diff
@@ -29,8 +29,32 @@ internal class HandleClickableUrl @Inject constructor(
                 val clickedEntry = knownDeeplinkActions.entries.firstOrNull { (key, _) ->
                     uriUtils.compareSchemeAuthorityAndPath(key, uri)
                 }
-                clickedEntry?.value?.invoke() ?: logger.error("Unrecognized clickable text: $uri")
+                clickedEntry?.value?.invoke(buildPayload(uri)) ?: logger.error("Unrecognized clickable text: $uri")
             }
         }
     }
+
+    private fun buildPayload(uri: String): DeeplinkPayload = DeeplinkPayload(
+        nextPaneOrDrawerOnSecondaryCta = uriUtils
+            .getQueryParameter(uri, ClickableLinkParameters.NEXT_PANE_OR_DRAWER_ON_SECONDARY_CTA)
+            ?.let { internalLinkToPaneId[it] }
+    )
+
+    //  this mapping is a bit unfortunate but this is what stripe.js decided to do:
+    private val internalLinkToPaneId = mapOf(
```

Comment:
This is sad but there's no other option - https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/4c7be206/src/linkedAccounts/shared-ui/v3/typography/serverTextParser.tsx#L38-L40